From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 29 Apr 2020 15:25:25 +0200
Subject: ANDROID: fscrypt: fix export symbol type

In commit d16d9c0046eb ("FROMLIST: fscrypt: Have filesystems handle their d_ops")
a new symbol was exported, but it should have been set as a _GPL symbol.

Fix this up by properly.

Bug: 138322712
Cc: Daniel Rosenberg <drosen@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I51e23761d3c898a0bf612048c8febd98a0817ae3
---
 fs/crypto/fname.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/crypto/fname.c b/fs/crypto/fname.c
--- a/fs/crypto/fname.c
+++ b/fs/crypto/fname.c
@@ -601,4 +601,4 @@ int fscrypt_d_revalidate(struct dentry *dentry, unsigned int flags)
 
 	return valid;
 }
-EXPORT_SYMBOL(fscrypt_d_revalidate);
+EXPORT_SYMBOL_GPL(fscrypt_d_revalidate);
